internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestVeniceHelixAdminWithSharedEnvironment.java

```diff
@@ -101,7 +101,6 @@ public class TestVeniceHelixAdminWithSharedEnvironment extends AbstractTestVenic
   @BeforeClass(alwaysRun = true)
   public void setUp() throws Exception {
     setupCluster(true, metricsRepository);
-    verifyParticipantMessageStoreSetup();
```

Comment:
Is this intended? can you help me understand why this is not needed anymore?

Reply:
Yes. `setupCluster` in `AbstractTestVeniceHelixAdmin` already calls this method internally. So, this was essentially a duplicate check